src/forms/FormFieldAddress/AddressAutocomplete/AddressAutocomplete.styled.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import theme from "@root/theme";
+import { StyledTextField } from "@root/forms/FormFieldText/FormFieldText.styled";
+
+// Components
+import ClearIcon from "@mui/icons-material/Clear";
+
+export const LocationSearchInputWrapper = styled.div`
+	&.mapCoordinates {
+		margin-bottom: 8px;
+		margin-top: 15px;
+	}
+`;
+
+export const StyledInputSearch = styled(StyledTextField)`
+  min-width: 684px;
+
+  &.MuiTextField-root {
+    display: flex;
+  }
+`;
+
+export const StyledClearIcon = styled(ClearIcon)`
+  cursor: pointer;
+  color: ${theme.newColors.almostBlack["100"]};
+`;
+
+export const SuggestionsContainer = styled.div`
```

Comment:
We should change the position of this component to absolute, that way the dropdown won't move all other elements around.